build.gradle.kts

```diff
@@ -36,7 +36,7 @@ dependencies {
   implementation("org.apache.commons:commons-csv:1.9.0")
 
   runtimeOnly("org.flywaydb:flyway-core")
-  runtimeOnly("org.postgresql:postgresql:42.3.1")
+  runtimeOnly("org.postgresql:postgresql:42.3.2")
```

Comment:
Updated version to fix this:
https://mojdt.slack.com/archives/C02DHRD2CE7/p1643786887065419

Reply:
Ah cool. I do a `./gradlew useLatestVersions` once in a while to grab the updates...